src/property.html

```diff
@@ -0,0 +1,1066 @@
+                      <p>Properties</p>
+                      <p>Testimonials</p>
+                      <p>FAQ’s</p>
+                    </div>
+                  </div>
+                  <div
+                    class="border-t border-solid border-color relative left-0 w-auto pr-3 sm:hidden"
+                  ></div>
+                </div>
+                <div
+                  class="flex flex-col sm:flex-row gap-3 sm:gap-10 xl:gap-14 border-r border-solid border-color sm:border-none pr-3 sm:pr-0"
+                >
+                  <div class="flex flex-col gap-4">
+                    <div class="font-medium text-base text-color-addition-info">
+                      Properties
+                    </div>
+                    <div
+                      class="flex flex-col gap-2 text-white font-medium text-sm"
+                    >
+                      <p>Portfolio</p>
+                      <p>Categories</p>
+                    </div>
+                  </div>
+                  <div
+                    class="border-t border-solid border-color relative left-0 w-auto sm:hidden"
+                  ></div>
+                  <div class="flex flex-col gap-4">
+                    <div class="font-medium text-base text-color-addition-info">
+                      Contact Us
+                    </div>
+                    <div
+                      class="flex flex-col gap-2 text-white font-medium text-sm"
+                    >
+                      <p>Contact Form</p>
+                      <p>Our Offices</p>
+                    </div>
+                  </div>
+                  <div
+                    class="border-t border-solid border-color relative left-0 w-auto sm:hidden"
+                  ></div>
+                </div>
+              </div>
+              <!-- second -->
+              <div
+                class="flex flex-col sm:flex-row gap-3 sm:gap-10 xl:gap-14 overflow-hidden"
+              >
+                <div
+                  class="flex flex-col gap-4 border-l border-solid border-color sm:border-none pl-6 sm:pl-0"
+                >
+                  <div class="font-medium text-base text-color-addition-info">
+                    About Us
+                  </div>
+                  <div
+                    class="flex flex-col gap-2 text-white font-medium text-sm"
+                  >
+                    <p>Our Story</p>
+                    <p>Our Works</p>
+                    <p>How It Works</p>
+                    <p>Our Team</p>
+                    <p>Our Clients</p>
+                  </div>
+                </div>
+                <div
+                  class="border-t border-solid border-color relative left-6 w-auto sm:hidden"
+                ></div>
+                <div class="flex flex-col gap-4 pl-6 sm:pl-0">
+                  <div class="font-medium text-base text-color-addition-info">
+                    Services
+                  </div>
+                  <div
```

Comment:
same as above

Reply:
i used li in

Reply:
again? please change them everywhere. As I said!!!

Reply:
means i change every div like <div>about</div> to <li>about</li>
actually my english is weak..

Reply:
![Uploading image.png…]()

Reply:
![image](https://github.com/user-attachments/assets/ae0e5475-daca-4a2b-b180-5578b87db801)